ui/src/app/components/runs/filters/datetime-range-filter/datetime-range-filter.component.ts

```diff
@@ -1,23 +1,23 @@
 /*
  * Copyright 2018 ABSA Group Limited
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Hmm did your IDE do that? That is very strange.

Reply:
Yes, it did. find and replace in the wrong selection. Fixed.